pkgs/development/python-modules/agate/default.nix

```
@@ -18,7 +18,7 @@
 
 buildPythonPackage rec {
   pname = "agate";
-  version = "1.7.1";
+  version = "1.9.1";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
The next time you see `format = setuptools`, please change it to `pyproject = true` to avoid legacy build.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters